src/components/TooltipButton/TooltipButton.tsx

```diff
@@ -26,7 +26,7 @@ export const TooltipButton = (props: TooltipButtonProps) => {
       <div className="tooltip-button__tooltip" aria-hidden>
         <span className="tooltip-button__tooltip-text">
           {props.label}
-          <span className="tooltip-button__hotkey">{` [${props.hotkeyKey}]`}</span>
+          {props.hotkeyKey !== undefined && <span className="tooltip-button__hotkey">{` [${props.hotkeyKey}]`}</span>}
```

Comment:
Just fyi, this could be written shorter:
```suggestion
          {props.hotkeyKey && <span className="tooltip-button__hotkey">{` [${props.hotkeyKey}]`}</span>}
```

But your way is more explicit and thus better to read 👍🏼